port/unix/omrintrospect.c

```diff
@@ -1685,7 +1681,7 @@ omrintrospect_threads_nextDo(J9ThreadWalkState *state)
 		goto cleanup;
 	}
 
-#if defined(LINUX) || defined(AIXPPC) || defined(J9ZOS390)
```

Comment:
Yes, that's right. The bug in the Linux scheduler was apparently fixed back in early 2009 and so the workaround should not be necessary.